app/controllers/users_controller.rb

```diff
@@ -0,0 +1,18 @@
+class UsersController < ApplicationController
+  def update
+    user = User.find_by(id: params[:id])
+    return render json: { errors: ['User not found'] }, status: :not_found unless user
+
+    if user.update(users_params)
+      render json: { user: user }, status: :ok
```

Comment:
I've been adding the success response inside a `data` attribute for the previously implemented APIs? Can we have the same pattern here as well to make it consistent with other APIs? I'm okay to have a discussion and change the other API response formats if you have any valid concerns around this response structure. Let me know